Mathlib/LinearAlgebra/TensorProduct/Associator.lean

```diff
@@ -256,6 +256,12 @@ variable {N}
 
 variable (g : P →ₗ[R] Q) (f : N →ₗ[R] P)
 
+local notation "α" => TensorProduct.assoc
+
+lemma lTensor_tensor (f : P →ₗ[R] Q) :
+    lTensor (M ⊗[R] N) f = (α R M N Q).symm ∘ₗ (f.lTensor N).lTensor M ∘ₗ α R M N P :=
```

Comment:
I don't think we should introduce local notation just to use it in one lemma.

Reply:
`open TensorProduct (assoc) in`